src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -664,7 +663,10 @@ export class BrowsingContextImpl {
         break;
     }
 
-    return {};
+    return {
+      navigation: this.navigableId ?? null,
```

Comment:
I think this in correct. We should set this to null if `BrowsingContext.ReadinessState.None` as now we will return the older one.

Reply:
I will change it to `null` for now which is better than returning the old ID, but I am not sure whether that's the final desired state.